.github/workflows/publish-to-pypi-test.yml

```diff
@@ -111,8 +109,29 @@ jobs:
         uses: tj-actions/changed-files@v2.1
 
       - name: Publish distribution 📦 to Test PyPI
-        if: github.ref == 'refs/heads/master'
         uses: pypa/gh-action-pypi-publish@master
         with:
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
           repository_url: https://test.pypi.org/legacy/
+        if: github.ref == 'refs/heads/master'
+
+  release:
+    runs-on: ubuntu-latest
+    needs: [build-n-publish]
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@master
+    - name: Create release
+      uses: Roang-zero1/github-create-release-action@master
+      with:
+        version_regex: ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Create GitHub release
+      uses: Roang-zero1/github-upload-release-artifacts-action@master
```

Comment:
This hasn't been updated since 2019, is there something else we can use that is better maintained?

Reply:
Apparently this one is still widely used. Considering it uses the release api which doesn't really change, it doesn't need active maintenance. I'll look further and see if I can find an actively maintained repo though.

Reply:
Thanks for checking.